README.md

```diff
@@ -1 +1,96 @@
-https://hub.docker.com/r/microsoft/oms/
+# Trying the container solution pack for Microsoft Operations Management Suite
+
+## What is the Microsoft Operations Management Suite?
+
+The Microsoft Operations Management Suite (OMS) is a software-as-a-service offering from Microsoft that allows Enterprise IT to manage any hybrid cloud. It offers log analytics, automation, backup and recovery, and security and compliance.  Sign up for a free account at [http://mms.microsoft.com](http://mms.microsoft.com) or read more about here: [https://www.microsoft.com/en-us/server-cloud/operations-management-suite/overview.aspx](https://www.microsoft.com/en-us/server-cloud/operations-management-suite/overview.aspx)
```

Comment:
These are still very long lines. This makes reviews later harder, make them 60-70 bytes/line (unless you can't due to a link, in which case the link should go on it's on line).